scenarioo-client/app/pipes/dateTime.pipe.ts

```diff
@@ -1,6 +1,8 @@
 import {Injectable, Pipe, PipeTransform} from '@angular/core';
 import {DatePipe} from '@angular/common';
 
+declare var angular: angular.IAngularStatic;
```

Comment:
please remove that - old AngularJS code must not be part of new components and I don't see any reference to it in the code